src/VisualStudio/Core/Test/DebuggerIntelliSense/CSharpDebuggerIntellisenseTests.vb

```diff
@@ -459,6 +459,162 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.DebuggerIntelliSense
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+$$</Document>
+                               <Document>class Program
+{
+    static void Main(string[] args)
+    [|{|]
```

Comment:
Oh, [we throw](http://source.roslyn.io/#Roslyn.Test.Utilities/MarkupTestFile.cs,171) if markup parsing doesn't work out, so I guess this works. Crazy.